osbs/tekton.py

```diff
@@ -544,11 +544,15 @@ def get_error_message(self):
                 plugin_errors = plugins_metadata.get('errors')
 
         err_message = ""
+        pipeline_error = None
 
         if plugin_errors:
             for plugin, error in plugin_errors.items():
                 err_message += f"Error in plugin {plugin}: {error};\n"
 
+        if 'message' in data['status']['conditions'][0]:
+            pipeline_error = data['status']['conditions'][0]['message']
```

Comment:
Can we use
```
pipeline_error = data['status']['conditions'][0].get('message')
```
here? we can avoid setting it to None and the condition.

Reply:
fixed